Vester/Tests/Host/ESXAdmins.Vester.ps1

```diff
@@ -0,0 +1,30 @@
+﻿# Test file for the Vester module - https://github.com/WahlNetwork/Vester
+# Called via Invoke-Pester VesterTemplate.Tests.ps1
+
+# Test title, e.g. 'DNS Servers'
+$Title = 'ESX Admins Group'
+
+# Test description: How New-VesterConfig explains this value to the user
+$Description = 'Security Group allowed root access to ESXi host'
+
+# The config entry stating the desired values
+$Desired = $cfg.host.esxadminsgroup
+
+# The test value's data type, to help with conversion: bool/string/int
+$Type = 'int'
```

Comment:
On my end, it looks like this test returns a `string`. Would you mind confirming?